contracts/modules/STO/USDTieredSTO.sol

```diff
@@ -104,7 +104,7 @@ contract USDTieredSTO is USDTieredSTOStorage, STO {
         uint256 _minimumInvestmentUSD,
         FundRaiseType[] memory _fundRaiseTypes,
         address payable _wallet,
-        address _reserveWallet,
+        address _treasuryWallet,
```

Comment:
Should we just remove this from configure similar to Dividend modules? Then allow them to override it in a subsequent call to changeAddresses if they want to?

Reply:
I keep it because of backward compatibility, but anyway I think we have to deploy everything for 3.0 release so I will make that change.